scripts/devel/build_and_test.py

```diff
@@ -73,6 +74,10 @@ def main(argv=sys.argv[1:]):
                 '-DBUILD_TESTING=1',
                 '-DCATKIN_ENABLE_TESTING=1', '-DCATKIN_SKIP_TESTING=0',
                 '-DCATKIN_TEST_RESULTS_DIR=%s' % test_results_dir]
+            if has_gpu_support():
+                ctest_args = ['-L "gpu_test"']
+            else:
+                ctest_args = ['-LE "gpu_test"']
```

Comment:
First, I would think the arguments should be two items in the list (`['-L', 'gpu_test']`) - not one item including a space?

Second, I don't understand how the presence of GPU support is supposed to toggle tests with/without the `gpu_test` label. How would you choose the behavior from the outside (e.g. from a build file)? How would you run all tests - including the GPU tests?

Reply:
> First, I would think the arguments should be two items in the list (`['-L', 'gpu_test']`) - not one item including a space?

Right, fixed here 7932b2f